Formatting Fixed

mgerweck opened the pull request:
This pull request is to display the movies in a clearer way. The movie title, tagline, and ID are displayed in a line across the website. 

Addresses issue #1

mgerweck committed:
formatting fixed

mgerweck committed:
feat: Formatting Fixed

This commit will change the formatting so the title, description, and ID are displayed for the user in a more clear way.

brandonijones commented:
This looks good, Michaela! I would suggest maybe adding comments to your template file explaining what it does.

mgerweck committed:
feat: Fix Formatting With Comments

This push will address issue #1 and has added comments to make the purpose more clear.
